### ISSUE TITLE
Highlighted options are not announced by screen readers

### ISSUE THREAD

**Issue by tillprochaska** [tillprochaska/accessible-select#5]
Add [`aria-activedescendant`](https://w3c.github.io/aria-practices/#kbd_focus_activedescendant).
